blueprint-integration/shared/src/main/java/com/github/dynamicextensionsalfresco/blueprint/DynamicExtensionsApplicationContext.java

```diff
@@ -156,7 +157,7 @@ protected void loadBeanDefinitions(@NotNull DefaultListableBeanFactory beanFacto
     @Override
     protected void initBeanDefinitionReader(@Nullable XmlBeanDefinitionReader beanDefinitionReader) {
         if (beanDefinitionReader == null) {
-            Intrinsics.throwNpe();
+            throw new IllegalArgumentException("beanDefinitionReader is null");
```

Comment:
The annotations says `beanDefinitionReader` can be `null` ?!

Reply:
This contradiction was already present in the original Kotlin code:
`    
override fun initBeanDefinitionReader(beanDefinitionReader: XmlBeanDefinitionReader?) {
        beanDefinitionReader!!.resourceLoader = this
        beanDefinitionReader.setNamespaceHandlerResolver(
                CompositeNamespaceHandlerResolver(
                        getOsgiNamespaceHandlerResolver(),
                        DefaultNamespaceHandlerResolver(classLoader
                ), getHostNamespaceHandlerResolver())
        )
        beanDefinitionReader.setEntityResolver(CompositeEntityResolver(getOsgiEntityResolver(), DelegatingEntityResolver(classLoader), getHostEntityResolver()))
    }
`
But it indeed doesn't make any sense.